client/lib/index.js

```diff
@@ -404,18 +406,6 @@ module.exports = class Client extends PassThrough {
 				this.log(error.stack);
 			})
 			.finally(async () => {
-				await new Promise((resolve, reject) => {
-					request
-						.get(`${this.uri.href}artifacts`)
-						.pipe(zlib.createGunzip())
-						.pipe(
-							fs.createWriteStream(
-								join(this.workdir, `${config.get('leviathan.artifacts')}.tar`),
-							),
-						)
-						.on('end', resolve)
-						.on('error', reject);
-				});
```

Comment:
We don't need to unzip here anymore?

Reply:
As far as I can tell, this wasn't even being used. We are getting the artifacts here instead https://github.com/balena-os/leviathan/blob/master/client/bin/multi-client.js#L143 , so this wasn't really doing anything. All it was doing was giving a misleading error if the testbot went offline mid test